### DISCUSSION TITLE
lxqt 1.0.0 in Fedora

### DISCUSSION THREAD

**Discussion by BluishHumility** [lxqt/lxqt#2125] [General]
I'm curious if there is any insight into when lxqt 1.0.0 might hit Fedora 35. I've been searching around since reading about the release but so far I haven't seen a timeline. The machine I have is still on 0.17.1.

**Comment by tsujan** [comment]
You may want to ask such questions in Fedora forums.

**Comment by marcosvile** [comment]
I use lximage on fedora lxqt34, and print it out with the command
$lximage-qt --screenshot

this command takes the print of an entire screen, which command do I use to capture only part of a screen?

**Reply by tsujan** [reply]
None.

`lximage-qt --help` says about `--screenshot`: "deprecated, please use screengrab instead". It means that lximage-qt isn't for taking screenshots, although that functionality isn't removed from it.

**Reply by marcosvile** [reply]
tks!
